runelite-client/src/main/java/net/runelite/client/plugins/objectindicators/ObjectIndicatorsConfig.java

```diff
@@ -67,9 +67,33 @@ default boolean highlightOutline()
 		return false;
 	}
 
+	@ConfigItem(
+			position = 2,
+			keyName = "highlightClickbox",
+			name = "Highlight Clickbox",
+			description = "Configures whether or not object should be highlighted by clickbox",
+			section = renderStyleSection
+	)
+	default boolean highlightClickbox()
+	{
+		return true;
```

Comment:
This is not a good default because you are defaulting both hull+clickbox on - which in addition to looking terrible, is also fairly slow, as we try to avoid doing clickbox calculation (by default?) at all if we can, due to it being slow. And is also a fairly jarring behavior change.